src/rulesets/oas2/index.json

```
@@ -179,6 +182,16 @@
       "tags": [
         "schema"
       ]
+    },
+    "oas2-unused-definition": {
+      "description": "Unused definition has been detected.",
```

I think we should rephrase it to 
```suggestion
      "description": "Potentially unused definition has been detected.",
```
These definitions might still be referenced by some other spec / file we have no clue of.

---

Fixed